std/haxe/coro/scopes/DefaultScopeComponent.hx

```diff
@@ -0,0 +1,66 @@
+package haxe.coro.scopes;
+
+import haxe.exceptions.CancellationException;
+import haxe.coro.coroutines.BaseCoroutine;
+
+@:access(haxe.coro.coroutines.BaseCoroutine)
+class DefaultScopeComponent extends ScopeComponent {
```

Comment:
I wonder if stateless components like this should be turned into a singleton. There's no reason to allocate them for each coroutine. The only question is where to put this and when/how to initialize it.